packages/ui/src/Table/Row.module.scss

```diff
@@ -18,18 +13,17 @@
       // Do not remove outline for high-contrast modes
       // https://stackoverflow.com/a/52616313
       outline: c.$outlineWidth solid transparent;
-      background-color: c.$colorInfoLight;
+      background-color: #e2e7fa;
```

Comment:
Maybe at least a local variable, since it's used several times

Reply:
Colors are not finalized yet. There is a separate task for that.